### DISCUSSION TITLE
Error installing pyg_lib under Torch version 2.1.0, CUDA version 11.8

### DISCUSSION THREAD

**Discussion by ghost** [pyg-team/pyg-lib#268] [Q&A]
Hi, I tried to install pyg_lib on Windows 10 OS but got the following error.

> pip install pyg_lib -f https://data.pyg.org/whl/torch-2.1.0+cu118.html
> ERROR: Could not find a version that satisfies the requirement pyg_lib (from versions: none)
> ERROR: No matching distribution found for pyg_lib

It seems that there is no package for windows that supports Torch version 2.1.0, CUDA 11.8.
As for torch_scatter, torch_sparse, torch_cluster and torch_spline_conv, I can install them without problems.
Can you please help me to solve this problem?
Thanks.

**Comment by rusty1s** [comment]
We currently do not provide pre-built Windows wheels, but
```
pip install git+https://github.com/pyg-team/pyg-lib.git
```
should (hopefully) work. Let me know.
